templates/dashboard/page/list.html

```diff
@@ -0,0 +1,86 @@
+{% extends "dashboard/base.html" %}
+{% load i18n %}
+{% load staticfiles %}
+{% load status %}
+{% load utils %}
+
+{% block title %}
+  {% trans "Pages" context "Pages list page title"%} - {{ block.super }}
+{% endblock %}
+
+{% block body_class %}body-page{% endblock %}
+
+{% block header_extra %}
+  {% if perms.page.edit_page %}
+    <div class="btn-fab-fixed">
+      <a class="btn-fab-default waves-effect waves-light"
+       href="{% url 'dashboard:page-add' %}"
+       title="{% trans "Add page page" %}"><svg data-src="{% static "dashboard/images/add.svg" %}" />
```

Comment:
No better way to say this to avoid this repetition?

Reply:
That's a mistake, it should say "Add page" (cc @akjanik). Also, translation context is missing.